src/Core/Math/MathExtented.js

```diff
@@ -78,4 +78,52 @@ MathExt.arrayRadToDeg = function(arrayDeg)
 	}
 };
 
+// TODO: Function in test :
+MathExt.step = function(val,stepVal)
+{
+    if(val<stepVal)
+        return 0.0;
+    else
+        return 1.0;
+
+};
+
+MathExt.exp2 = function(expo)
+{
+    return Math.pow(2,expo);
+};
+
+MathExt.parseFloat2= function(str) {
```

Comment:
This function is used a single time, with a constant value (`this.parseFloat2(0x800000);`) : let's remove this function, and replace the call by the value.
Btw, this function seems to expect a `string` and you pass it a number ?!